walnuts_cpp/CMakeLists.txt

```diff
@@ -67,6 +68,26 @@ if (WALNUTS_BUILD_EXAMPLES)
   target_compile_options(test_nuts PRIVATE -O3 -Wall)
 endif()
 
+option(WALNUTS_BUILD_STAN "Build Stan example with BridgeStan" OFF)
+set(WALNUTS_BRIDGESTAN_TAG "1d4ff6a10d572ee22ed26b0274059bb4fe885187" CACHE STRING "Version of BridgeStan to use")
```

Comment:
Should this be an option as well?

Reply:
I don't think so, for the same reason Eigen's version isn't further up -- changing it might require changes to our code here for it to still all work together